sources/Renderer/Direct3D11/Buffer/D3D11Buffer.cpp

```diff
@@ -430,6 +430,12 @@ void D3D11Buffer::WriteWithSubresourceCopyWithCpuAccess(
     WriteWithStagingBuffer(context, stagingBuffer.Get(), data, dataSize, dstOffset);
 }
 
+void D3D11Buffer::SetVertexAttribs(const ArrayView<VertexAttribute>& vertexAttribs)
+{
+    if (!desc.vertexAttribs.empty())
```

Comment:
This is the typo that causes the Windows CIS builds to fail: `desc.` must be removed here.